darwin/dataset/remote_dataset.py

```diff
@@ -260,11 +260,13 @@ def get_report(self, granularity="day"):
             raw=True,
         ).text
 
-    def release(self, name: Optional[str] = None):
+    def release(self, annotation_class_ids: Optional[List] = None, name: Optional[str] = None):
```

Comment:
I don't think name is optional anymore. @andreaazzini ?

Reply:
it is not below, but for the python function it is. In fact, if it is None then empty list is provided to the payload (I simulated a requested and inspected with the networking tab)